svelte-jest-legacy.js

```diff
@@ -4,5 +4,9 @@ module.exports = {
         'jest/globals': true
     },
     extends: ['./svelte-base-legacy', 'plugin:jest/recommended'],
-    plugins: ['jest']
+    plugins: ['jest'],
+    rules: {
+        'jest/expect-expect': ['off'],
+        'jest/no-done-callback': ['off'],
```

Comment:
Off because why be limited to one style of writing async tests (just like any other Javascript). [[docs](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-done-callback.md)]
Also the `done` style is still prominent in [Jest docs](https://jestjs.io/docs/asynchronous#callbacks).